Update comet.de_DE.po

martinschaible opened the pull request:
Consolidating phrases and strings
Continuing with translationg

martinschaible committed:
Update comet.de_DE.po

Consolidating phrases and strings
Continuing with translationg

mason-cometbackup commented:
Thank you for the update! These changes are great.

This particular change is still a work-in-progress. For instance, the `Speicherziel` change is mentioned but not yet completed. I will delay applying these changes into our own copy.

You can continue to make changes to your copy of the file via the `patch-3` version on your own account at https://github.com/martinschaible/translations/blob/patch-3/comet.de_DE.po : any changes to your copy of the file will be reflected in this "Pull Request", and show up as additional commits here.

Once the bulk change is made, we can merge the "Pull Request" into our own copy of the file, so that it will appear in a future version of Comet,

martinschaible committed:
Update comet.de_DE.po

martinschaible committed:
Update comet.de_DE.po

martinschaible committed:
Update comet.de_DE.po

martinschaible committed:
Update comet.de_DE.po

martinschaible committed:
Update comet.de_DE.po

- Strings replaced for "snapshot" and "Storage Vault"
- Typos fixed
- New translations

martinschaible committed:
Update comet.de_DE.po

Recheck the work from yesterday: Typos fixed

martinschaible committed:
Update comet.de_DE.po

- New translations
- Typos fixed

martinschaible committed:
Update comet.de_DE.po

- Additional translations

mason-cometbackup commented:
Fantastic work! We will prepare to merge these current changes into the upcoming 19.11.0 release.
